CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## unreleased
+
+  - Add `vaultedPaymentMethodTypesThatShouldBeHidden` option to `dropin.create`
```

Comment:
I'm not super happy with this name for the param, but I also can't think of a better name that describes what it does.

@braintree/team-sdk any of y'all have ideas for this?